### PR TITLE
Add favicon to skeleton template

### PR DESCRIPTION
**Pull request by sidharthv96**
build was failing for the default skeleton template as favicon was missing.

I chose to add the favicon inside static rather than removing the reference from app.html as it would be helpful to know where to place static files and reference them.

### Before submitting the PR, please make sure you do the following
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts

### CODE REVIEW

**Comment by benmccann** [comment]
seems reasonable like a reasonable fix. could you add a changeset as well so that this will be able to be released?

**Comment by sidharthv96** [comment]
@benmccann Done!

**Review by benmccann — APPROVED** [review]

File: packages/kit/src/runtime/client/router.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ export class Router {
 
 			if (incorrect) {
 				info.path = has_trailing_slash ? info.path.slice(0, -1) : info.path + '/';
-				history.replaceState({}, '', `${info.path}${location.search}`);
+				history.replaceState({}, '', `${this.base}${info.path}${location.search}`);
```
Comment (benmccann) [inline packages/kit/src/runtime/client/router.js:245]: Can you drop this change from this PR? It's unrelated so should be its own PR

Comment (sidharthv96) [reply packages/kit/src/runtime/client/router.js:245]: Oops, I was fixing another issue, my bad.
